doc/administrator/mapfile.rst

```diff
@@ -406,6 +406,29 @@ Performance improvement
 Adding an ``EXTENT`` parameter to the ``LAYER`` section may significantly improve the performances
 because it saves MapServer from computing the extent of all layer features.
 
+Generating shapefile as a raster tileindex
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you need raster tileindex (e.g. from a .TIF field picture) you can generate a
+corresponding shapfile by doing these steps:
```

Comment:
To have good performance you should have tiled files with overview, and probably a tileindex, you can ...